### PR TITLE
feat(hedera): added support for hedera network

### PR DESCRIPTION
**Pull request by AnithaKrypC**
added mainnet and testnet hedera network configurations.

### CODE REVIEW

**Comment by AnithaKrypC** [comment]
@isaackps  Please review this PR

File: tests/fixtures/v3/document-hedera.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      "type": "d0edac0b-dc8f-4f88-ba35-1dd768b8a758:string:EMBEDDED_RENDERER",
+      "url": "922c1295-9826-4fa1-99e1-5faa41e59db9:string:https://tutorial-renderer.openattestation.com"
+    },
+    "recipient": {
+      "name": "f7ca97ee-af7c-40a2-886a-df1311428ce8:string:KrypC - Piruthivi"
+    },
+    "issuers": [
+      {
+        "name": "0b7facd6-d192-43f4-8d0d-bae22b3ec43d:string:KrypC Issuer",
+        "tokenRegistry": "daafd0da-521b-4961-a92d-88328b9c7790:string:0x3DE43bfd3D771931E46CbBd4EDE0D3d95C85f81A",
```
[the remark this answers is not in the document]
Comment (AnithaKrypC) [reply tests/fixtures/v3/document-hedera.json]: @isaackps changed to V3

File: tests/fixtures/v3/document-hedera-no-network.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    "id": "352cee06-38a9-4e2f-bbcd-d656dfc6bcde:string:1111",
+    "date": "fe252099-59a5-4a8f-a7ad-a38262d92099:string:2018-02-21",
+    "customerId": "9378dd68-cfab-4420-9565-c6de94d76a0c:string:564",
+    "terms": "d490c2e8-f3ac-4302-97d0-b5f082965b56:string:Due Upon Receipt",
+    "billFrom": {
+      "name": "8c69940e-c464-491f-abde-a3f3f983214d:string:ABC Company",
+      "streetAddress": "2a466cdc-ddad-4b19-9e18-bcbbab67b7d6:string:Level 1, Industry Offices",
+      "city": "a1675df0-adae-47c1-9c93-53bcb0a01355:string:Singapore",
+      "postalCode": "025029af-1198-4e88-9f65-247a76058e42:string:123456",
+      "phoneNumber": "ae8d8c1d-193e-4440-a115-f4eb823998ea:string:60305029"
```
[the remark this answers is not in the document]
Comment (AnithaKrypC) [reply tests/fixtures/v3/document-hedera-no-network.json]: @isaackps changed to V3

**Comment by isaackps** [comment]
Please fix this

**Comment by AnithaKrypC** [comment]
> Please fix this

@isaackps updated node version now

**Review by isaackps — APPROVED** [review]

**Pull request merged**